projects/frontend/data-pipelines/gui/projects/data-pipelines/src/lib/components/data-jobs-explore/components/grid/data-jobs-explore-grid.component.ts

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright 2021-2023 VMware, Inc.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { Component, ElementRef, Inject, Input, OnInit } from '@angular/core';
+import { DOCUMENT, Location } from '@angular/common';
+import { ActivatedRoute, Router } from '@angular/router';
+
+import { ComponentService, ErrorHandlerService, NavigationService, RouterService } from '@vdk/shared';
+
+import { DATA_PIPELINES_CONFIGS, DataPipelinesConfig, DisplayMode } from '../../../../model';
+import { DataJobsApiService, DataJobsService } from '../../../../services';
+
+import { DataJobsBaseGridComponent } from '../../../base-grid/data-jobs-base-grid.component';
+
+@Component({
+    selector: 'lib-data-jobs-explore-grid',
+    templateUrl: './data-jobs-explore-grid.component.html',
+    styleUrls: ['./data-jobs-explore-grid.component.scss']
+})
+export class DataJobsExploreGridComponent extends DataJobsBaseGridComponent implements OnInit {
+    //Decorators are not inherited in Angular. If we need @Input() we need to declare it here
+    @Input() override teamNameFilter: string;
+    @Input() override displayMode: DisplayMode;
+
+    readonly uuid = 'DataJobsExploreGridComponent';
+
+    constructor( // NOSONAR
```

Comment:
Nit: This comment can be removed. We're not using Sonar, right?

Reply:
Sonar already got commented on
* https://github.com/vmware/versatile-data-kit/blob/main/specs/vep-1507-vdk-operations-ui/README.md#e2e-test-image
* https://github.com/vmware/versatile-data-kit/blob/1a66e6ff273c74c626497914d8afa28c05e34f59/projects/frontend/cicd/Dockerfile#L39

Adding a note to cleanup in the corresponding task we have in todo